package.json

```diff
@@ -45,10 +45,10 @@
     "ea-start-prod-db": "./build.js -run --watch --settings ../ForumCredentials/settings-local-dev-prod-db.json --mongoUrlFile ../ForumCredentials/prod-db-conn.txt --postgresUrlFile ../ForumCredentials/prod-pg-conn.txt",
     "ea-start-testing-db": "./build.js -run --watch --settings ./settings-test.json --mongoUrl ${TESTING_DB_URL:=$(cat ../ForumCredentials/testing-db-conn.txt)} --postgresUrl ${PG_URL:=$(cat ../ForumCredentials/testing-pg-conn.txt)}",
     "init-cypress-db": "yarn ea-command 'Globals.dropAndCreatePg({templateId: \"cypress_template\", seed: true, dropExisting: true})'",
-    "init-cypress-db-ci": "yarn ea-command-ci 'Globals.dropAndCreatePg({templateId: \"cypress_template\", seed: true, dropExisting: true})'",
+    "init-cypress-db-ci": "./scripts/timeout.sh 120 yarn ea-command-ci 'Globals.dropAndCreatePg({templateId: \"cypress_template\", seed: true, dropExisting: true})'",
```

Comment:
Whilst I was fixing cypress bugs, this failed a couple of times in a way that meant the server wasn't able to close properly and it just kept running indefinitely until I checked back on the CI run and realised it never ended. This will kill it after two minutes (which should be way more time than it actually needs).